.gitmodules

```diff
@@ -75,12 +90,12 @@
 [submodule "external/vita-toolchain"]
 	path = external/vita-toolchain
 	url = https://github.com/Vita3K/vita-toolchain
-[submodule "external/yaml-cpp"]
-	path = external/yaml-cpp
-	url = https://github.com/jbeder/yaml-cpp
 [submodule "external/VulkanMemoryAllocator"]
 	path = external/VulkanMemoryAllocator
 	url = https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator.git
-[submodule "external/ffmpeg"]
-	path = external/ffmpeg
-	url = https://github.com/Vita3K/ffmpeg-core.git
+[submodule "external/yaml-cpp"]
+	path = external/yaml-cpp
+	url = https://github.com/jbeder/yaml-cpp
+[submodule "external/zlib"]
+	path = external/zlib
+	url = https://github.com/korewawatchful/zlib.git
```

Comment:
This is like a million new submodules. I think we might be able to get some to work on the main repo.

While this is way better than the 200k line solution, I'm not sure if I'm comfortable merging this yet.